### PR TITLE
Add missing parentheses

### PR DESCRIPTION
**Pull request by francas**
Documentation content update for [Module 3-cosmjs-adv, Section 2-cosmjs-messages]

This PR adds missing parentheses to sample code.

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [x] Small language/grammar fixes
* [ ] Small content fixes 
* [ ] Addition of new content
* [x] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### CODE REVIEW

File: hands-on-exercise/3-cosmjs-adv/2-cosmjs-messages.md [hunk 1]
```diff
@@ -635,7 +635,7 @@ it("can continue the game up to before the double capture", async function () {
         r: await getShortAccountInfo(bob),
     }
     // TODO
-}
+});
```
Comment (xavierlepretre) [inline hands-on-exercise/3-cosmjs-adv/2-cosmjs-messages.md:638]:
```suggestion
})
```

The linked code does not use semicolons.

File: hands-on-exercise/3-cosmjs-adv/2-cosmjs-messages.md [hunk 2]
```diff
@@ -740,7 +740,7 @@ it("can send a double capture move", async function () {
     const firstCaptureMove: GameMove = completeGame[24]
     const secondCaptureMove: GameMove = completeGame[25]
     // TODO
-}
+});
```
Comment (xavierlepretre) [inline hands-on-exercise/3-cosmjs-adv/2-cosmjs-messages.md:743]:
```suggestion
})
```

**Review by xavierlepretre — APPROVED** [review]

**Review by coldice — APPROVED** [review]

File: hands-on-exercise/3-cosmjs-adv/2-cosmjs-messages.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ Make sure you have all you need before proceeding:
 
 * You understand the concepts of [CosmJS](/tutorials/7-cosmjs/1-cosmjs-intro.md).
 * You have generated the necessary TypeScript types in [the previous tutorial](./1-cosmjs-objects.md).
-* You have the finished the checkers blockchain exercise. If not, you can follow that tutorial [here](/hands-on-exercise/1-ignite-cli/index.md) or just clone and checkout the [relevant branch](https://github.com/cosmos/b9-checkers-academy-draft/tree/wager-denomination) that contains the final version.
+* You have the finished checkers blockchain exercise. If not, you can follow that tutorial [here](/hands-on-exercise/1-ignite-cli/index.md) or just clone and checkout the [relevant branch](https://github.com/cosmos/b9-checkers-academy-draft/tree/v1-wager-denomination) that contains the final version.
```
Comment (xavierlepretre) [inline hands-on-exercise/3-cosmjs-adv/2-cosmjs-messages.md:18]: Wait no, it should not be v1

Comment (xavierlepretre) [reply hands-on-exercise/3-cosmjs-adv/2-cosmjs-messages.md:18]: Fixed here https://github.com/cosmos/sdk-tutorials/pull/1339/files#diff-78cf44d53f0679346a9032b2596d806dcf762eba69dc4bfc7d6ff17265d6c71b

**Pull request merged**